play-by-ear.md

```diff
@@ -0,0 +1,51 @@
+# How to synthesise attestation results produced by Veraison
+
+## Prerequisite
+
+### Step CLI
+
+The [`step` CLI](https://smallstep.com/cli/) has an awesome `crypto jwt` subcommand to deal with JWTs (of which EAR is an instance).
+
+See [installation instructions](https://smallstep.com/docs/step-cli/installation) for the most common platforms.
+
+(TL;DR -- On MacOSX, do `brew install step`.)
+
+### Obtain the `veraison/ear` examples
+
+* Clone Veraison's EAR repo:
+```sh
+git clone github.com/veraison/ear
```

Comment:
I don't think this works exactly as written without the explicit `https://` on the front of the GitHub URL, at least it didn't for me, but possibly it depends on what shell you're using.

Reply:
Thanks for spotting this!

Reply:
Fixed in [a2ad0fd](https://github.com/CCC-Attestation/attested-tls-poc/pull/22/commits/a2ad0fda4ae59862a385aef644da74649a52ce73)